src/Application/DefaultPreset.php

```diff
@@ -49,11 +50,16 @@ public static function get(): array
                 UnusedVariableSniff::class => [
                     'ignoreUnusedValuesWhenOnlyKeysAreUsedInForeach' => true,
                 ],
+                PropertyTypeHintSniff::class => [
+                    'enableNativeTypeHint' => $composer->hasPhpVersion() ? $composer->lowestPhpVersionIsGreaterThenOrEqualTo('7.4') === true : null,
+                ],
             ],
         ];
+
+        return array_filter($config);
```

Comment:
Is this `array_filter` really needed ? 

I think PropertyTypeHintSniff will be still here. 

Just do a quick test in php:

```
<?php

$arr = [
    'test' => 'true',
    'property' => ['test' => false]
];
    
var_dump($arr);
var_dump(array_filter($arr));
```

It's return the same array
```
array(2) { ["test"]=> string(4) "true" ["property"]=> array(1) { ["test"]=> bool(false) } }
array(2) { ["test"]=> string(4) "true" ["property"]=> array(1) { ["test"]=> bool(false) } }
```

Reply:
Yeah, good point, was a mistake from my side :) Fixed it.